build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 plugins {
     java
-    id("com.github.hierynomus.license")
-    id("com.github.sgtsilvio.gradle.utf8")
```

Comment:
we don't need the utf8 plugin anymore?

Reply:
this is replaced with the `defaults` plugin, which includes everything from the `utf8` plugin + some more settings that should be default, but are not because of backwards compatibility.